judge/models/problem.py

```diff
@@ -535,3 +558,38 @@ class Meta:
         )
         verbose_name = _('solution')
         verbose_name_plural = _('solutions')
+
+
+class ProblemPointsVote(models.Model):
+    points = models.IntegerField(
+        verbose_name=_('proposed points'),
+        help_text=_('the amount of points you think this problem deserves.'),
```

Comment:
Is this help text visible to the user? If it's just admins seeing this help text, using `you` here feels a bit weird.